pkgs/top-level/all-packages.nix

```diff
@@ -9093,17 +9093,15 @@ let
   neard = callPackage ../servers/neard { };
 
   nginx = callPackage ../servers/http/nginx {
-    rtmp        = true;
-    fullWebDAV  = true;
-    syslog      = false; # the patch is not found
-    moreheaders = true;
+    modules = [ nginxModules.rtmp nginxModules.dav nginxModules.moreheaders ];
```

Comment:
If so, I'd leave a comment about it. Someone a month from now will come
across it and instinctively refactor to be the more obvious thing. This
feels like a lurking trap :(

On Mon, Oct 19, 2015 at 09:37 Nikolay Amiantov <notifications@github.com>
wrote:

> In pkgs/top-level/all-packages.nix
> <https://github.com/NixOS/nixpkgs/pull/10474#discussion_r42370984>:
>
> > @@ -9093,17 +9093,15 @@ let
> >    neard = callPackage ../servers/neard { };
> >
> >    nginx = callPackage ../servers/http/nginx {
> > -    rtmp        = true;
> > -    fullWebDAV  = true;
> > -    syslog      = false; # the patch is not found
> > -    moreheaders = true;
> > +    modules = [ nginxModules.rtmp nginxModules.dav nginxModules.moreheaders ];
>
> I can't find any reference on this, but if I'm right w.r.t. precedence of
> rec over with I think it's better to leave this as is to make it more
> future-proof, otherwise some package added later would have a chance to
> spontaneously break unrelated things.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/10474/files#r42370984>.
>